custom-commands/hospitality/deployment/README.md

```diff
@@ -127,11 +127,31 @@ az group delete --name MyResourceGroupName
 
 *This section needs to be updated.*
 
-The resources were created using an azure template which is stored in the [./azuredeploy.json](./azuredeploy.json) file.</br>
-Then the [../storage files](../storage-files) were deployed into the storage resource.</br>
+First an Azure resource group was created using the resourceName parameter as the name.
+
+Then resources were created using an azure template which is stored in the [./azuredeploy.json](./azuredeploy.json) file. Parameters were passed in from the powershell script by manipulation the json values in the [./azuredeploy.parameters.json](./azuredeploy.parameters.json) file. For the individual resources a random number between 0 and 9999 is attached to help guarantee availability.</br>
```

Comment:
a four digit random number between 0000 and 9999

Reply:
nope we don't pad. I could change that, but it doesn't pad with extra 0's right now

Reply:
ah okay, that explains why I thought I remember you had 3 digits before :-) better to have consistent file name templates as I document the pattern  MyResouceGroupName-####. But not urgent. Fix that post //build (please open issue for it)